src/pip/_internal/utils/logging.py

```diff
@@ -186,6 +196,11 @@ def setup_logging(verbosity, no_color, user_log_file):
                 "()": IndentingFormatter,
                 "format": "%(message)s",
             },
+            "indent_and_timestamp": {
```

Comment:
`indent_with_timestamp` would be a little better (since timestamp is an argument to the indenting formatter).

Reply:
Done.